Sources/MapboxNavigation/NavigationMapViewDelegate.swift

```diff
@@ -40,6 +40,7 @@ public protocol NavigationMapViewDelegate: AnyObject, UnimplementedLogging {
      */
     func navigationMapView(_ navigationMapView: NavigationMapView, routeCasingLineLayerWithIdentifier identifier: String, sourceIdentifier: String) -> LineLayer?
     
+    
```

Comment:
Extra line

Reply:
Thanks for the suggestion, the typo and extra line are updated.